### DISCUSSION TITLE
How to out put the force and motion data of a elastomer object?

### DISCUSSION THREAD

**Discussion by Eric-Yuki** [DualSPHysics/DualSPHysics#269] [DUALSPHYSICS_V5.2]
I am simulating the impact of an elastic body on the oscillation of a floating body under liquid sloshing. However, I can currently only observe the motion of the floating body. I would like to further examine the changes in the elastic body, such as its deformation and the forces it experiences over time. 

          
                    <setmkbound mk="1" />
                    <drawbox>
                        <boxfill>solid</boxfill>
                        <point x="-11" y="-2.5" z="5" />
                        <size x="0.5" y="5" z="0.5" />
                    </drawbox>
                    <drawbox>
                        <boxfill>solid</boxfill>
                        <point x="10.5" y="-2.5" z="5" />
                        <size x="0.5" y="5" z="0.5" />
                    </drawbox>
                    <drawbox>
                        <boxfill>solid</boxfill>
                        <point x="-2.5" y="5.5" z="5" />
                        <size x="5" y="0.5" z="0.5" />
                    </drawbox>
                    <drawbox>
                        <boxfill>solid</boxfill>
                        <point x="-2.5" y="-6" z="5" />
                        <size x="5" y="0.5" z="0.5" />
                    </drawbox>
                    <shapeout file="bougen"/>
                </mainlist>
            </commands>
        </geometry>
        <motion>
            <objreal ref="1">
                <begin mov="1" start="0" />
                <mvnull id="1" />
            </objreal>
        </motion>
		<floatings>
            <floating mkbound="10">
            <massbody value="200000" />
                        <center x="0" y="0" z="4.5" />
            </floating>		
		</floatings>	
    </casedef>
    <execution>
        <special>
            <flexstrucs>
                <flexstrucbody mkbound="1" mkclamp="0">
                    <density value="1000.0" comment="Mass density" units_comment="kg/m^3" />
                    <youngmod value="1.4e6" comment="Young Modulus" units_comment="Pa" />
                    <poissratio value="0.45" comment="Poisson ratio" />
                    <constitmodel value="1" comment="Constitutive model 1:Plane Strain (2D), 2:Plane Stress (2D), 3:St. Venant Kirchhoff (3D)" />
                    <hgfactor value="0.0" comment="Hourglass correction factor: keep as low as possible (default=0)" />
                </flexstrucbody>
            </flexstrucs>
            <accinputs>
                <accinput mkfluid="0">
                    <acccentre x="0" y="0" z="2" comment="Center of acceleration" units_comment="metres (m)" />
                    <globalgravity value="0" comment="Global gravity enabled (1) or disabled (0)" />
                    <acctimesfile value="CaseSloshingAccData.csv" comment="File with linear and angular acceleration data" />
                </accinput>
            </accinputs>

**Comment by alexbexe** [comment]
You should contact here Dr Joseph O’Connor. The University of Edinburgh, UK: 
joseph j.oconnor@epcc.ed.ac.uk

**Comment by JMY123JMY** [comment]
your question is solved? I have same questions
